### DISCUSSION TITLE
[Script Request] Switch between "Standart View" and "Stacked View"

### DISCUSSION THREAD

**Discussion by ulahcherubim** [windingwind/zotero-actions-tags#288] [Ideas]
I want to use main page in stacked view and reader tabs in standart view. There is no way to seperate them and switch easily between modes. If there was a shortcut, it would save me some clicks everytime I switch between them.

**Comment by liangtao** [comment]
if you use macos, you can customize your shortcuts of standard/stacked view, system settings -> keyboard ->  keyboard shortcuts -> App shortcuts, then selecting Zotero application and setting shortcuts of Menu title, **Standard View, Stacked View** repectively. Please make sure that the big/small letters of menu titles are correct.
